api/all/src/main/java/io/opentelemetry/api/trace/Span.java

```diff
@@ -81,9 +81,6 @@ static Span wrap(SpanContext spanContext) {
       ApiUsageLogger.log("context is null");
       return getInvalid();
     }
-    if (!spanContext.isValid()) {
```

Comment:
This seems to make sense to me, but the javadoc above somewhat implies that this behavior:

> It will not be exported and all tracing operations are no-op, but it can be used to propagate a valid {@link SpanContext} downstream.

Can we remove the word "valid"?

Also, you could argue this is breaking behavior change, but I don't think so. If you provide an invalid span context (i.e. invalid span id / trace id) but useful TraceFlags / TraceState, you'll still get a Span with an invalid span context back. It would be odd for someone to be relying on the behavior of `Span.wrap(SpanContext` clearing effectively clearing TraceFlags / TraceState when span id and trace id are invalid.

Reply:
Yes, I fully agree. Furthermore, I think that using _valid_ or _invalid_ with respect to the `SpanContext` is a bit misleading. The context represents an _empty_ trace (i.e. with no spans yet), and every trace starts from such a context.